syft/frameworks/crypten/context.py

```diff
@@ -102,9 +103,14 @@ def wrapper(*args, **kwargs):
             world_size = len(workers) + 1
             return_values = {rank: None for rank in range(world_size)}
 
+
+            hook_plan_building()
```

Comment:
This is needed such that when we build the plan, we use another set of methods (currently only ```load``` is changed).

Reply:
you can also put this as a comment in the code I'd say